test/common/helm_install.go

```diff
@@ -0,0 +1,62 @@
+// Package common contains utility functions for all the tests in this repo
+package common
+
+import (
+	"crypto/tls"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+)
+
+// NamespaceName : common namespace variable to use in the tests
+var NamespaceName = "marklogic-" + strings.ToLower(random.UniqueId())
+
+// KubectlOpt : common kubectl options variable to use in the tests
+var KubectlOpt = k8s.NewKubectlOptions("", "", NamespaceName)
+
+// HelmInstallFn : common function to install helm chart for e2e tests
+func HelmInstallFn(t *testing.T, options map[string]string, releaseName string) string {
+	// Path to the helm chart we will test
+	helmChartPath, e := filepath.Abs("../../charts")
+	if e != nil {
+		t.Fatalf(e.Error())
+	}
+	t.Logf("====Creating namespace: " + NamespaceName)
+	k8s.CreateNamespace(t, KubectlOpt, NamespaceName)
+
+	helmOptions := &helm.Options{
+		KubectlOptions: KubectlOpt,
+		SetValues:      options,
+	}
+	t.Logf("====Installing Helm Chart")
+	helm.Install(t, helmOptions, helmChartPath, releaseName)
+
+	tlsConfig := tls.Config{}
+	podName := releaseName + "-marklogic-0"
+	// wait until the pod is in Ready status
+	k8s.WaitUntilPodAvailable(t, KubectlOpt, podName, 10, 15*time.Second)
+	tunnel7997 := k8s.NewTunnel(KubectlOpt, k8s.ResourceTypePod, podName, 7997, 7997)
```

Comment:
Remove between line 45 to line 61. As a install function. Testing 7997 connectivity is not necessary. Move these code back to install_test, as these needs to be test just once.

Reply:
removed